### PR TITLE
rustc_mir: remove extra space when pretty-printing MIR.

### PR DESCRIPTION
**Pull request by anyska**

### CODE REVIEW

File: src/test/mir-opt/retag.rs [hunk 1]
```diff
@@ -114,7 +114,7 @@ fn main() {
 // }
 // END rustc.main-{{closure}}.EraseRegions.after.mir
 // START rustc.ptr-drop_in_place.Test.SimplifyCfg-make_shim.after.mir
-// fn  std::intrinsics::drop_in_place(_1: *mut Test) -> () {
```
Comment (oli-obk) [inline src/test/mir-opt/retag.rs]: I don't get how this is the only test affected. Shouldn't https://github.com/anyska/rust/blob/7d400199c2b1778ba7fd4748a5cb16d926a85d67/src/test/mir-opt/inline/inline-closure-borrows-arg.rs#L20 have been printing the space before this PR, too?

Comment (oli-obk) [reply src/test/mir-opt/retag.rs]: also I just saw that my linked thing is missing a space in the end... Maybe MIR opt tests ignore whitespaces even in the middle of things?

Comment (eddyb) [reply src/test/mir-opt/retag.rs]: > Maybe MIR opt tests ignore whitespaces even in the middle of things?

We were confused by this too, I think that's the case, and I'd rather it not.

Comment (oli-obk) [reply src/test/mir-opt/retag.rs]: If we move loads of mir-opt tests to the [blessable system](https://github.com/rust-lang/rust/pull/69916) the leftover cases that still want to use the manual system should be few enough for that to be doable in a painless way.

**Comment by oli-obk** [comment]
@bors r+ rollup

Not testable right now, but improves mir printing output

**Pull request merged**